src/commands/cont.rs

```diff
@@ -47,3 +66,61 @@ impl ContinueCommand {
         Ok(())
     }
 }
+
+fn get_time_to_continue(
+    time_entries: Vec<TimeEntry>,
+    running_time_entry: Option<TimeEntry>,
+    interactive: bool,
+) -> Option<TimeEntry> {
+    match interactive {
+        true => get_time_entry_from_user(time_entries),
+        false => get_first_stopped_time_entry(time_entries, running_time_entry),
+    }
+}
+
+fn get_first_stopped_time_entry(
+    time_entries: Vec<TimeEntry>,
+    running_time_entry: Option<TimeEntry>,
+) -> Option<TimeEntry> {
+    // Don't continue a running entry that was just stopped.
+    let continue_entry_index = match running_time_entry {
+        None => 0,
+        _ => 1,
+    };
+    return time_entries.get(continue_entry_index).cloned();
+}
+
+fn get_time_entry_from_user(time_entries: Vec<TimeEntry>) -> Option<TimeEntry> {
+    let options = SkimOptionsBuilder::default()
+        .height(Some("100%"))
```

Comment:
Why 100%? I think it looks quite neat to have it appear as a popup in parts of the screen. I'm not super adamant on it, so if you think 100 is better and can prove it I can buy it

Reply:
I choose 100% cause it's an arbitary choice for now, no other number made more sense.
We could go with 30% but that'll only limit the number of time entries visible to the user.
Also Skim draws over the entire screen anyway so that extra space is wasted.

Reply:
Then let's leave it at 100%